src/mscorlib/shared/System/Runtime/CompilerServices/ValueTaskAwaiter.cs

```diff
@@ -38,5 +38,8 @@ public struct ValueTaskAwaiter<TResult> : ICriticalNotifyCompletion
 
         /// <summary>Gets the task underlying <see cref="_value"/>.</summary>
         internal Task<TResult> AsTask() => _value.AsTask();
+
+        /// <summary>Gets the task underlying <see cref="_value"/>.</summary>
+        public Task GetTask() => _value.AsTask();
```

Comment:
This should be an explicit interface implementation so it doesn't show up in the public surface area:
```C#
Task IValueTaskAwaiter.GetTask() => _value.AsTask();
```

Reply:
Will fix.